operators/src/source/gdal_source.rs

```diff
@@ -416,15 +431,8 @@ where
         let tiling_strategy = self.tiling_specification.strategy(x_signed, y_signed);
 
         stream::iter(tiling_strategy.tile_information_iterator(query.bbox))
-            .map(move |tile| (tile, info.clone()))
-            .then(async move |(tile, info)| {
-                Ok(RasterTile2D::new_with_tile_info(
-                    info.time,
-                    tile,
-                    Self::load_tile_data_async(info.params.clone(), tile).await?,
-                ))
-            })
-            .boxed()
+            .map(move |tile| Self::load_tile_async(info.params.clone(), tile, info.time))
+            .buffered(4) // TODO: find a good default and / or add to config.
```

Comment:
Either remove buffering or leave a buffer of one with a comment.
But 4 is too arbitrary for me.

Reply:
changed it to 1